package-lock.json

```diff
@@ -2817,9 +2817,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30001252",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001252.tgz",
-      "integrity": "sha512-I56jhWDGMtdILQORdusxBOH+Nl/KgQSdDmpJezYddnAkVOmnoU8zwjTV9xAjMIYxr0iPreEAVylCGcmHCjfaOw==",
+      "version": "1.0.30001399",
```

Comment:
do we want to commit that ?

Reply:
So the npm is complaining we are using outdated database and if you fix it it bumps `caniuse` which is fine